Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json

```diff
@@ -0,0 +1,563 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000537",
+    "checksum": "271d71c4faf2a2b323cd4fb125849e415c2b0feb96eb1cd9a93720068ade23ac",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Single-cell RNA sequencing identifies extracellular matrix gene expression by pancreatic circulating tumor cells",
+        "version": "1.0.0",
+        "license": "",
+        "created": "2019-10-08T21:22:14.871482",
+        "modified": "2019-10-17T05:25:51.188495",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "2019-10-16"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Nirmala Ramnarine",
+                "orcid": "https://orcid.org/0000-0002-9001-8153",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "nramnarine13@gwu.edu"
+            },
+            {
+                "name": "David T. Ting",
+                "orcid": "https://orcid.org/0000-0002-3261-2322",
+                "affiliation": "Massachusetts General Hospital Cancer Center, MASSACHUSETTS GENERAL HOSPITAL CANCER CENTER, Dana-Farber/Harvard Cancer Center, Massachusetts General Hospital, Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112